activemodel/lib/active_model/type/boolean.rb

```diff
@@ -1,9 +1,22 @@
 module ActiveModel
   module Type
-    class Boolean < Value # :nodoc:
+    # == Active \Model \Type \Boolean
+    #
+    # A class representing +boolean+ with functionality for coercion to and from
+    # a database or user input.
+    #
+    # === Coercion
+    # Values set from user or database input will first be coerced into a
+    # appropriate ruby type.
+    #
+    # - Values specified in +FALSE_VALUES+ will be coerced to +false+.
+    #   - e.g. "false", "f" , "0", and +0+ will all be coerced to +false+
```

Comment:
I'd prefer this to be a bit more human. Something like:

`false`, `"f"`, `"0"`, `0`, or any other value in `FALSE_VALUES` will be coerced to `false`